server.py

```diff
@@ -38,6 +38,22 @@ def add_note():
 
     return jsonify(response)
 
+@app.route('/notes/edit/<id>', methods=['POST'])
+def update_edited_note_in_BD(id):
+    """ Replace Note in Database by ID with Edited Note and Title """
+
+    note_id = request.form.get('note_id')
+    note_title = request.form.get('note_title')
+    note_content = request.form.get('note_content')
+
+    
+    note = update_note(note_id, note_title, note_content)
+    print "update is running RUNRUNRUNRUN", note.content, note.created_at
+
+    flash("Your Edits Have Been Saved")
+
+    return redirect('/')
```

Comment:
At the moment is does post a message and follows the redirect to / but not properly as it is repeated with 3xx response. I will look into this.